### DISCUSSION TITLE
Unstar text is awkward

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49621] [Mobile] [answered]
GitHub Mobile beta

1. Visit https://github.com/dns-stats/visualizer
2. Tap ⭐
3. Add to list
4. Tap ⭐

![Screenshot_20230309-081341.png](https://user-images.githubusercontent.com/2119212/224034072-fab5e84b-b8ac-49d2-8e58-736548463fe5.png)

```diff
-Unstarring this repo will remove it from your lists 
+Unstarring this repo will also remove it from your lists 
```

The current dialog text almost implies it will not unstar it, and instead *only* remove it from lists.

**Comment by stevepopovich** [comment, marked as answer]
Hi! Thanks for the feedback, I passed it on to the team for prioritization.
